src/pages/docs/packaging-applications/build-servers/github-actions.md

```diff
@@ -1,557 +1,131 @@
-:::
-
-:::div{.success}
-**Example GitHub Actions Repo:**
-
-View a working GitHub Actions examples on our [samples GitHub repository](https://github.com/OctopusSamples).
-- [.NET Core](https://github.com/OctopusSamples/OctoPetShop/blob/master/.github/workflows/dotnet-core.yml)
-- [Java using Maven](https://github.com/OctopusSamples/RandomQuotes-Java/blob/master/.github/workflows/maven.yml)
-:::
-
-## GitHub Action Integrations v3
-
-GitHub Action Integrations v3 no longer requires the Octo CLI to function. This new native approach brings many benefits to the table, including but not limited too:
-- Smaller actions
-- Faster runtimes
-- Performance benefits through the [Executions API](https://octopus.com/blog/faster-deployments-with-the-executions-api)
-
-### Migration Guides
-
-Migrations guides can be found on each [GitHub Action Integration Repo](https://github.com/OctopusDeploy/create-release-action/blob/main/migration-guide.md).
-
-### New GitHub Actions
-
-The following GitHub actions have been created in collaboration with Github Action v3.
-
-### Deploy Release
-
-Incorporate the following actions in your workflow to deploy a release in Octopus Deploy using an API key, a target instance (i.e. server), and a project:
-
-```yaml
-env:
-
-steps:
-  # ...
-  - name: Deploy a release in Octopus Deploy 🐙
-    uses: OctopusDeploy/deploy-release-action@v3
-    env:
-      OCTOPUS_API_KEY: ${{ secrets.API_KEY  }}
-      OCTOPUS_URL: ${{ secrets.SERVER }}
-      OCTOPUS_SPACE: 'Outer Space'
-    with:
-      project: 'MyProject'
-      release_number: '1.0.0'
-      environments: |
-        Dev
-        Test
-      variables: |
-        Foo: Bar
-        Fizz: Buzz
+        project: 'MyProject'
+        release_number: '1.0.0'
+        environments: |
+            Dev
+            Test
+        variables: |
+            Flip: Bling
+            Fizz: Buzz
+        git_ref: ${{ github.ref }}
+        git_commit: ${{ github.sha }}
 ```
 
-### Deploy Release Tenanted
+### ✍️ Environment Variables
 
-```yaml
-env:
+| Name              | Description                                                                                                                                              |
+| :---------------- | :------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `OCTOPUS_SPACE`   | The Name of the Space where this command will be executed.                                                                                               |
+| `OCTOPUS_URL`     | The base URL hosting Octopus Deploy (i.e. `https://octopus.example.app`). It is strongly recommended that this value retrieved from a [GitHub secret.](https://docs.github.com/en/actions/security-for-github-actions/security-guides/using-secrets-in-github-actions) |
```

Comment:
Does this value actually need to be a secret? Most of our examples recommend it, but then our OICD login action shows the URL in the example in text. Wasn't sure which to go with here.